app/components/admin/users/tools/notes_component.rb

```diff
@@ -0,0 +1,12 @@
+module Admin
+  module Users
+    module Tools
+      class NotesComponent < ViewComponent::Base
+        def initialize(user:)
+          @user = user
+          @notes = user.notes.includes(:author).order(created_at: :desc).limit(10)
```

Comment:
This is totally unrelated to this PR but I've been wondering for a long time if we have to many AR chains in our controllers/services and should start leveraging named scopes more. Something like

```ruby
@notes = Notes.most_recent_for(user: user, limit: 10)
```
reads much friendlier.

Reply:
I agree but I think I'll leave it like this for now, in the sense that I would like to have something like that being a focused PR that changes names and refactors the code, so that it becomes intentional for the entire team

Reply:
Yes, sorry, this was one of my tangents, not a suggestion to address this here. But this certainly seems like something we could slowly introduce during downtimes, one long AR method chain at a time 😃